[2024-04-16 10:01:00,657] INFO _log() L96    * Restarting with stat - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:12,454] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:12] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:12,484] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:12] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:12,715] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:12] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:12,793] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:12] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:13,137] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:13] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:13,167] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:13] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:01:59,817] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:01:59] "GET /valvula/sonda?sonda=NSXX HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:02:17,008] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:02:17] "GET /valvula/sonda?sonda=NSYY HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:02:27,037] ERROR log_exception() L828  Exception on /valvula/sonda [GET] - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py L828 
Traceback (most recent call last):
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_openapi3/scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/slsn/repos/bop_land_api/blueprints/valvula.py", line 65, in get_bop
    bop_id = session.query(BOP).filter(BOP.sonda == bop_sonda).first().id
AttributeError: 'NoneType' object has no attribute 'id'
[2024-04-16 10:02:27,038] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:02:27] "[35m[1mGET /valvula/sonda?sonda=aaaaaaaa HTTP/1.1[0m" 500 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:19,559] INFO _log() L96    * Detected change in '/home/slsn/repos/bop_land_api/blueprints/valvula.py', reloading - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:19,697] INFO _log() L96    * Restarting with stat - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:28,821] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:28] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:28,841] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:28] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:29,084] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:29] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:29,145] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:29] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:29,517] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:29] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:29,544] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:29] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:12:59,155] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:12:59] "[31m[1mGET /valvula/sonda?sonda=NSXX HTTP/1.1[0m" 401 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:08,539] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:08] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:08,549] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:08] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:08,804] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:08] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:08,852] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:08] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:09,196] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:09] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:09,197] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:09] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:13:24,209] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:13:24] "POST /auth/login HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:14:00,237] INFO _log() L96   127.0.0.1 - - [16/Apr/2024 10:14:00] "GET /valvula/sonda?sonda=NSXX HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-16 10:14:09,246] ERROR log_exception() L828  Exception on /valvula/sonda [GET] - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py L828 
Traceback (most recent call last):
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_openapi3/scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/slsn/repos/bop_land_api/blueprints/valvula.py", line 65, in get_bop
    bop_id = session.query(BOP).filter(BOP.sonda == bop_sonda).first().id
AttributeError: 'NoneType' object has no attribute 'id'
[2024-04-16 10:14:55,884] INFO _log() L96    * Restarting with stat - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
